cpu/esp32/doc.txt

```diff
@@ -1027,31 +1027,46 @@ be used as ADC channels. See section [GPIO pins](#esp32_gpio_pins) and
 Several definitions can be used during compile time to configure the
 _Light-sleep_ and the _Deep-sleep_ mode:
 
+<center>
+
+Parameter        | Default                  | Mode  | Description
+:----------------|:-------------------------|:------|:------------
+ESP_PM_GPIO_HOLD | not defined              | Deep  | Hold GPIO output level if defined
+ESP_PM_WUP_PINS  | none                     | Deep  | GPIOs used as wake-up source
+ESP_PM_WUP_LEVEL | ESP_PM_WUP_PINS_ANY_HIGH | Deep  | Level for wake-up pins to wake-up
+ESP_PM_WUP_UART0 | disabled                 | Light | Positive UART0 RxD signal edges to wake-up
+ESP_PM_WUP_UART1 | disabled                 | Light | Positive UART1 RxD signal edges to wake-up
+
+</center>
+
+- If `ESP_PM_GPIO_HOLD` is defined, GPIOs hold their last output level
+  when entering _Deep-sleep_ mode. Please note that only RTC GPIOs
+  can hold their output value in _Deep-sleep_ mode.
```

Comment:
Can you please also update the [GPIO table](https://github.com/RIOT-OS/RIOT/blob/f744cb063f6907b93def85469dc3eceb8cd51dce/cpu/esp32/doc.txt#L516) header

```patch
--- a/cpu/esp32/doc.txt
+++ b/cpu/esp32/doc.txt
@@ -511,7 +511,7 @@ ESP32 has 34 GPIO pins, where only a subset can be used as output, as ADC channe
 
 <center>
 
-Pin    | Type   | ADC | PU/PD  | Special function | Remarks
+Pin    | Type   | ADC / RTC | PU/PD  | Special function | Remarks
 -------|--------|:---:| ------ |------------------|--------
 GPIO0  | In/Out | yes | yes    | Touch sensor | Bootstrapping, pulled up
 GPIO1  | In/Out | -   | yes    | UART0 TxD    | Console
```